chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.ts

```diff
@@ -65,21 +66,17 @@ export class CustomerSummaryComponent implements OnInit {
   public deleteRow(targetCustomer) {
     let targetCustomerIndex;
     if (targetCustomer !== null && targetCustomer !== undefined) {
-      /*if (this.customers.includes(customer)) {
-          const customerIndex = this.customers.indexOf(customer);
-          this.customers.splice(customerIndex, 1);
-        }*/
-      this.customers.forEach((customer, customerIndex) => {
+      this.customerService.customers.forEach((customer, customerIndex) => {
         if (customer.id === targetCustomer.id) {
           targetCustomerIndex = customerIndex;
         }
       });
-      this.customers.splice(targetCustomerIndex, 1);
+      this.customerService.customers.splice(targetCustomerIndex, 1);
     }
   }
 
   public addRow() {
-    const customer = this.customers;
+    const customer = this.customerService.customers;
```

Comment:
No need to create a variable here when you can just use the `this.customerService.customers` array directly.  Plus the variable name you create is singular and not plural.  Array variables need to be plural.

Reply:
Removed variable and re-applied directly into line.